### DISCUSSION TITLE
Why does attention alignment always stay below a level (VITS model)?

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [coqui-ai/TTS#2018] [General Q&A] [answered]
Hi,

I don't understand why during training of the VITS model the alignment graph shown in tensorboard does not go above a level. No matter which dataset I tried, or up to which iteration step I go, it never reaches the upper part of the graph although the test audios are very decent.
![alignment40k](https://user-images.githubusercontent.com/91517923/192196196-06c1c19b-9ade-466a-aa75-c83ca6c3959a.png)

Is this graph a good metrics for VITS or it does not matter actually ?

Could someone shed a light on this fact that has been bugging me for weeks :-D

**Comment by lexkoro** [comment, marked as answer]
I think it actually did reach the end. 
The empty space might be padding, which is why it is not displayed/ relevant.
